extract_data/metadata.py

```diff
@@ -728,6 +728,61 @@ def add_missing_references_to_paper_collection():
                         paper["pubmed_id"]["value"],
                     )
 
+def manually_add_paper(dict):
```

Comment:
Note that `dict` is also the name of something defined by default in Python (namely, the type object for dictionaries)... so this is hiding our ability to use the type object in this function.